packages/db-migrations/databases/fxa/patches/patch-137-136.sql

```diff
@@ -0,0 +1,21 @@
+-- -- Delete new row from table
+-- DELETE FROM securityEventNames WHERE name='account.login.failure';
+-- DELETE FROM securityEventNames WHERE name='account.two_factor_added';
+-- DELETE FROM securityEventNames WHERE name='account.two_factor_requested';
```

Comment:
Interesting, I think we have 17 delete lines but only 16 insert lines. I think my quick (yet unreliable) eyeballs spotted `'account.two_factor_requested'` as possibly being deleted but not inserted.